extensions/resteasy-reactive/quarkus-resteasy-reactive-servlet/runtime/src/main/java/io/quarkus/resteasy/reactive/server/servlet/runtime/ServletRequestContext.java

```diff
@@ -574,4 +575,26 @@ public V setValue(V value) {
             return old;
         }
     }
+
+    @Override
+    public ServerHttpResponse sendFile(String path, long offset, long length) {
+        context.response().sendFile(path, offset, length);
```

Comment:
This bypasses a lot of Servlet stuff, it really needs to write the file using the Servlet API's.

Reply:
Are there any specific APIs for servlet to write files?